src/templates/drizzle/labelheader.hbs

```diff
@@ -0,0 +1,24 @@
+<{{tag}} {{#if id}}id="{{id}}"{{/if}} class="
+  drizzle-u-inlineBlock
+  drizzle-u-alignMiddle
+  drizzle-u-textNoWrap
+  drizzle-u-padRight">
+  {{#if key}}
```

Comment:
@mrgerardorodriguez I think we can simplify it. 

If we supply an `id`, then we can use that to determine whether or not the anchor is output.

```hbs
{{#if id}}
  <a href="#{{id}}">{{> @partial-block}}</a>
{{else}}
  {{> @partial-block}}
{{/if}}
```

```hbs
{{#> drizzle.labelheader id=(toSlug data.title)}}
  {{data.title}}
{{/drizzle.labelheader}}
```